operators/README.md

```diff
@@ -71,10 +78,22 @@ The Instance Operator implements the backend logic necessary to spawn new enviro
 - **Instance** defines an instance of a certain template. The manipulation of those objects triggers the reconciliation logic in the operator, which creates/destroy associated resources (e.g.; Virtual Machines).
 
 ### Persistent Feature
-The Instance Operator allows you to create persistent Virtual Machines. Persistent means that the VM can be stopped and restarted, deleted and recreated without data loss. To make this possible, the content of the VM's disk is stored in a PVC.
+
+The Instance Operator allows you to create persistent VirtualMachines (VM). Persistent means that the VM can be stopped and restarted, deleted and recreated without data loss. 
 In order to use this feature, [the CDI operator](../infrastructure/virtualization/README.md) has to be installed.
-Granted the CDI operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which version to create is performed looking at the corresponding field in the `template` spec.
-N.B. The process of creating a persistent VirtualMachine can take a bit more time with the respect to a normal one (5-10 mins, depending on the size of the image). However when you recreate the VM you will not have to wait such that time.
+
+Provided that the CDI Operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which kind of VM is wanted, i.e. persistent or not, is performed looking at the corresponding field in the `template` spec. In the same way you can choose the size of the persistent disk that the VM will have.
+
+It has been decided to use a VM instead of directly creating a VirtualMachine Instance (VMI) because the start and stop process is already managed by KubeVirt. In this way the VMI is easily stopped and started again modifying the running field of Instance resource (this can be easily done from the user interface).
+
+It is important to notice that the Instance Operator has to watch not only the Instance but also the DataVolume associated to it. In other words when an Instance or a DataVolume is modified the controller is triggered and it starts a reconcile in order to achieve a consistent state in the cluster. Since only one specific change in the DataVolume is important (the status Phase becoming Succeeded) and a lot of changes in the Instance do not need controller actions (those in the status) a  [Predicate](https://sdk.operatorframework.io/docs/building-operators/golang/references/event-filtering/) has been used. That function acts before the reconcile, working as a filter and letting the reconcile function be called only in the case explained before.
```

Comment:
```suggestion
It is important to notice that the Instance Operator has to watch not only Instance resources but also the DataVolumes associated to them. In other words when an Instance or a DataVolume is modified, the controller is triggered and it starts a reconcile process in order to achieve a consistent state in the cluster. Since only one specific change in the DataVolume is important (i.e., when the status Phase becoming Succeeded) and a lot of changes in the Instance do not need controller actions (e.g., those affecting the status) a [Predicate](https://sdk.operatorframework.io/docs/building-operators/golang/references/event-filtering/) has been used. That function acts before the reconcile, working as a filter and letting the reconcile function be called only in the cases explained before.
```